src/globalConfig/accounts.ts

```diff
@@ -36,3 +38,26 @@ export async function promptForEmailUnverified() {
     source: 'promptForEmailUnverified',
   });
 }
+
+export async function checkEmailStatusOrExit() {
+  const email = getUserEmail();
+  try {
+    const resp = await fetchWithTimeout(
+      `https://api.promptfoo.app/api/users/status?email=${email}`,
+      undefined,
+      500,
+    );
+    const data = (await resp.json()) as {
+      status: 'ok' | 'exceeded_limit';
+      error?: string;
+    };
+    if (data?.status === 'exceeded_limit') {
+      logger.error(
+        'You have exceeded the maximum cloud inference limit. Please contact inquiries@promptfoo.dev to upgrade your account.',
+      );
+      process.exit(1);
+    }
+  } catch (e) {
+    logger.debug('Failed to check user status', e);
```

Comment:
logger only takes one argument

```suggestion
    logger.debug(`Failed to check user status: ${e}`);
```